xpti/doc/SYCL_Tracing_Implementation.md

```diff
@@ -58,16 +58,15 @@ submission. This may be as simple as adding a new node to the task-graph or
 adding multiple nodes to the graph, where one of the nodes represents the
 computation and the others dependent memory transfers.
 
-To model this, we create a global graph for every application instantiation
-and all kernel executions in the applications are added as nodes in this
-global graph. In the SYCL runtime, there is no obvious location where the
-creation of the global graph can be inserted as many objects are
-instantiated statically. Currently, we embed the graph creation in the
-plugin interface (PI) layer `initialize()` call. In this call, we will
-perform two operations:
+To model this, we create a global graph for every application instantiation and
+all kernel executions in the applications are added as nodes in this global
+graph. In the SYCL runtime, there is no obvious location where the creation of
+the global graph can be inserted as many objects are instantiated statically.
+Currently, we perform the graph creation during UR initialization, in the
+`initializePlugins()` call. In this call, we will perform two operations:
```

Comment:
Do you plan to rename `initializePlugins` to something like `initializeBackends`?

Reply:
I think that should come in with the "rename `Plugin`" change, right now the function technically is initializing a list of `Plugin` objects